extensions/amp-form/0.1/test/test-utils.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isFieldDefault} from '../utils';
+
+describes.realWin('amp-form-utils', {}, env => {
+  let doc;
+
+  beforeEach(() => {
+    doc = env.win.document;
+  });
+
+  describe('isFieldDefault', () => {
+    it("returns if a text field's value matches its default value", () => {
+      const html = '<input type="text" value="default">';
```

Comment:
Elements are inserted as HTML so that the `defaultValue` property is generated correctly.